NetWork.md

```diff
@@ -0,0 +1,142 @@
+# The (dwyl) NetWork [_Work-in-Progress_]
+
+A network of creative technology people engaged
```

Comment:
Link to https://github.com/dwyl/phase-two/blob/1183967dcc448cee3abd264f4f43d6b0ed5725ac/braga/README.md#what-is-a-creative-technologist-